src/master-playlist-controller.js

```diff
@@ -329,6 +294,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
         type: 'mediachange',
         bubbles: true
       });
+      this.trigger('mediachange');
```

Comment:
at first I thought we could watch for it in hls and then update video.js audiotracks. but it turned out that that was the wrong event. I will remove this line good catch.